File: C++/problem0709.cpp
#include <iostream>
#include <iomanip>
#include <vector>
#include <map>
#include <set>
#include <algorithm>
#include <cmath>
#include <ctime>
#include <cstdlib>
#include <windows.h>

using namespace std;

class Solution {
public:
    string toLowerCase(string str) {
        for (int k = 0; k < str.size(); ++k)
        {
            // if ('A'<=str[k] && str[k]<='Z')
            if (65<=str[k] && str[k]<=90)
            {
                str[k] += 32;
            }
        }
        return str;
    }
};

int main()
{
    string str = "Hello";
    Solution solu;
    string ans = solu.toLowerCase(str);
    cout << ans << endl;
    return 0;
}

File: Python/problem0709.py
# -*- coding: utf-8 -*-
"""
Created on Tue May 28 17:18:44 2019

@author: Administrator
"""

class Solution:
    def toLowerCase(self, string: str) -> str:
#        return string.lower()
        string = list(string)
        for k in range(len(string)):
            if 'A'<=string[k] and string[k]<='Z':
                temp = ord(string[k])
                temp += 32#chr(ord('1')+3)
                string[k] = chr(temp)
        return ''.join(string)
        
        
        
        
        
solu = Solution()
string = 'Hello'
string = 'here'
string = 'LOVELY'
print(solu.toLowerCase(string))